client/tests/components/form/OperationsForm.test.tsx

```diff
@@ -1,184 +1,173 @@
-  opt_in: null,
-  operator: 1,
-  status: "Not Started",
-  regulated_products: [],
-  reporting_activities: [],
-  contacts: [],
-  operator_id: 1,
+  opt_in: false,
+  regulated_products: [1],
+  previous_year_attributable_emissions: null,
+  status: "Draft",
   naics_code_id: 1,
-  current_year_estimated_emissions: null,
+  first_name: "John",
+  last_name: "Doe",
+  email: "john.doe@example.com",
+  phone_number: "+16044011234",
+  position_title: "Senior Officer",
+  street_address: "123 Main St",
+  municipality: "Cityville",
+  province: "ON",
+  postal_code: "A1B 2C3",
+  statutory_declaration: "data:text/plain;base64,SGVsbG8gV29ybGQ=",
+  bc_obps_regulated_operation: null,
+  bcghg_id: "23219990001",
+  external_point_of_contact_first_name: "John",
+  external_point_of_contact_last_name: "Doe",
+  external_point_of_contact_email: "john.doe@example.com",
+  external_point_of_contact_phone_number: "+16044011234",
+  external_point_of_contact_position_title: "Senior Officer",
+  "Did you submit a GHG emissions report for reporting year 2022?": false,
+  is_external_point_of_contact: true,
+  multiple_operators_array: [{}],
 };
 
+const testOperationSchema = createOperationSchema(
+  operationSchema,
+  [
+    {
+      id: 1,
+      naics_code: "211110",
+      naics_description: "Oil and gas extraction (except oil sands)",
+    },
+  ],
+  [
+    { id: 1, name: "BC-specific refinery complexity throughput" },
+    { id: 2, name: "Cement equivalent" },
+    { id: 3, name: "Chemicals: pure hydrogen peroxide" },
+  ],
+  [],
+);
+
 describe("Operations component", () => {
   beforeEach(() => {
-    jest.restoreAllMocks();
+    vi.clearAllMocks();
   });
-  it("renders the empty OperationsForm when no formData is passed", async () => {
-    render(<OperationsForm schema={operationSchema} />);
-
-    // Test for Legend elements
-    expect(
-      screen.getByText(/Step 1: Operation Information/i),
-    ).toBeInTheDocument();
-    expect(
-      screen.getByText(
-        /Step 2: Operation Operator Information - If operation has multiple operators/i,
-      ),
-    ).toBeInTheDocument();
-    expect(
-      screen.getByText(/Step 3: Operation Representative \(OR\) Information/i),
-    ).toBeInTheDocument();
 
-    /* Test for Input elements */
+  it("renders the empty OperationsForm when no formData is passed", async () => {
+    useRouter.mockReturnValue({
+      query: { operation: "create", formSection: "1" },
+      replace: vi.fn(),
+    } as Router);
+    useParams.mockReturnValue({
+      formSection: "1",
+      operation: "create",
+    } as QueryParams);
+
+    render(<OperationsForm schema={testOperationSchema} formData={{}} />);
+
+    // Test for the Operation Information form heaer
+    expect(screen.getByTestId("field-template-label")).toBeVisible();
     // Operation Name
-    expect(screen.getByLabelText(/Operation Name+/i)).toBeInTheDocument();
+    expect(screen.getByLabelText(/Operation Name+/i)).toHaveValue("");
```

Comment:
Why are we expecting "" as a value? Wouldn't that mean it isn't showing up correctly?

Reply:
This is the `renders the empty OperationsForm when no formData is passed` test. Using `toBeNull()` returns and error, same with `toHaveValue(undefined)`. Though the undefined value works on plain text fields, I think it's likely something with how some of the MUI widgets handle empty fields.